practice-app/website/templates/create_event.html

```diff
@@ -39,7 +40,7 @@
 
             if (response.status == 201) {
                 response.json().then(event_data => {
-                    window.location.assign('/event/' + event_data["id"])
+                    window.location.assign('/event/' + event_data["id"] + "?token={{token}}")
```

Comment:
Insecure redirect with accesible auth token in URI

Reply:
I have updated the token system to store the tokens in a Flask session instead of the URL.